torch/utils/_sympy/value_ranges.py

```diff
@@ -632,6 +635,11 @@ def where(a, b, c):
         b = ValueRanges.wrap(b)
         c = ValueRanges.wrap(c)
         a = a.boolify()
+        # Could be removed once we track dtypes properly when setting unknown in loads/reductions etc.
+        if a == ValueRanges.unknown():
+            return b
+        if b == ValueRanges.unknown():
+            return a
```

Comment:
This makes no sense? Isn't `a` a completely different dtype from `b` and `c`.

Reply:
Ugh, right, this doesn't make sense at all...